sky/packages/sky/lib/src/widgets/scrollable.dart

```diff
@@ -216,14 +238,18 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
     _animation.stop();
   }
 
+  void _handleDragStart() {
+    dispatchOnScrollStart();
+  }
+
   void _handleDragUpdate(double delta) {
     // We negate the delta here because a positive scroll offset moves the
     // the content up (or to the left) rather than down (or the right).
     scrollBy(-delta);
   }
 
-  void _handleDragEnd(Offset velocity) {
-    fling(velocity);
+  Future _handleDragEnd(Offset velocity) {
+    return fling(velocity).then((_) { dispatchOnScrollEnd(); });
```

Comment:
what if the fling is canceled by the user? will dispatchOnScrollEnd() still be called?